tests/plugins/toolbar/manual/aria-haspopup.md

```diff
@@ -0,0 +1,18 @@
+@bender-tags: 4.10.0, feature, 2072
```

Comment:
I don't think that we have any other test with `-` in its name. Let's rename it to `ariahaspopup`